Branch/BranchQRCode.m

```diff
@@ -112,7 +112,7 @@ - (void) callQRCodeAPI:(NSDictionary*_Nullable)params
             completion:(void(^)(NSData * _Nullable qrCode, NSError * _Nullable error))completion {
     
     NSError *error;
-    NSString *branchAPIURL = [[BranchConfiguration new] branchAPIServiceURL];
+    NSString *branchAPIURL = [[BranchConfiguration init] branchAPIServiceURL];
```

Comment:
Calling `init` without allocating memory won't have any effect. It might actually crash in runtime because it cannot init a class object.

But `BranchConfiguration`'s `init` is marked as `NS_UNAVAILABLE`, so we have to use the designed initializer: `-[BranchConfiguration initWithKey:]`.

```suggestion
    NSString *branchAPIURL = [[[BranchConfiguration alloc] initWithKey:@""] branchAPIServiceURL];
```

Since `BranchQRCode` doesn't actually need the configuration, just the base URL which's specified there, I initialized the Branch Configuration without a key. Again, since the only thing needed is the base URL, it doesn't matter if the key is set or not (which was the reason why `new` was used in the in the first place without the key).

Hope it helps.